shunting-yard-binary/src/main/resources/shunting-yard-minimal.yml.template

```diff
@@ -9,6 +9,8 @@ event-receiver:
   configuration-properties:
     com.hotels.shunting.yard.event.receiver.sqs.queue: https://sqs.us-west-2.amazonaws.com/1234567/sqs-queue
     com.hotels.shunting.yard.event.receiver.sqs.wait.time.seconds: 15
+    # max.messages acceptable values: 1 to 10, default is 10. For more details, see https://docs.aws.amazon.com/AWSSimpleQueueService/latest/APIReference/API_ReceiveMessage.html
```

Comment:
Is this something you noticed was missing while going through this?

Reply:
This is something that I've added. `SqsMessageReader.Builder.withMaxMessages()` lets you configure the maximum number of messages returned when you poll a queue. So I've added the config property here, along with a short description of how to use it.

Do you think this is sufficient or should I document this elsewhere?

Reply:
This is fine, it's just that we're now mixing two very different things in one PR - a new feature and a big removal of code. I'd generally prefer these as two separate PRs, which is a bit more work but I think makes things much easier to review.

Reply:
OK, if we stick with this PR for now then we need a line in the changelog describing this new feature.

Reply:
I am happy to remove this instead to keep this PR clean, and we can add this in later on? I don't mind, up to you :)

Reply:
If it's not too much work and is possible to split them into two prs that would be nice - one does the move "as is" with hopefully no changes other than re-packaging and a few other small necessary changes and the other introduces this new feature?

Reply:
Done, removed from this PR